lighthouse-core/test/results/sample_v2.json

```diff
@@ -2118,27 +2068,24 @@
       "score": 0.88,
       "scoreDisplayMode": "numeric",
       "rawValue": 150,
-      "displayValue": [
-        "Potential savings of %d KB",
-        63
-      ],
+      "displayValue": "Potential savings of 63 KB",
       "details": {
         "type": "opportunity",
         "headings": [
           {
             "key": "url",
             "valueType": "url",
-            "label": "Uncompressed resource URL"
```

Comment:
I think it's better to have this full label
otherwise i could interpret this table to include already gzipped stuff

Reply:
aw I was actually excited by the consistency now

It seems like the same argument could be made for all the other audits though. `Large Image URL`, `Uncached URL`, `Unused Image URL`, that all seems a bit much is there something unique about the compression audit?

Reply:
`Large Image URL`, `Uncached URL`, `Unused Image URL` sounds awesome. :)

wdyt?

Reply:
😭 that's not the direction I wanted this to go, lol

I think it's a bit redundant given the audit title and full description are immediately above the column names. And as @brendankenny as noted, i18n makes me want to consolidate the strings we use not aggressively expand them 😆